components/content-service-api/typescript/src/initializer_grpc_pb.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
```

Comment:
@mustard-mh Could you try to remove the copyright date changes from the PR? Ideally those are never touched after the file got created, plus it spams the diff. :pray: